contrib/test/ci/test.yml

```diff
@@ -64,9 +68,12 @@
       poll: 30
 
     - name: run integration tests with userNS enabled + alternate results file
-      shell: "make localintegration 2>&1 | tee /tmp/artifacts/testout.txt"
+      shell: | 
+        set -eou pipefail
+        make localintegration 2>&1 | tee /tmp/artifacts/testout.txt
```

Comment:
this should be a different filename so it doesn't stomp on the integration tests in the previous block.

Reply:
make localintegration 2>&1 > /tmp/artifacts/testout.txt

and

make localintegration 2>&1 > /tmp/artifacts/test_with_userns.txt [or something like that]